doc/man7/EVP_ASYM_CIPHER-SM2.pod

```diff
@@ -0,0 +1,41 @@
+=pod
+
+=head1 NAME
+
+EVP_ASYM_CIPHER-SM2
```

Comment:
Hmm I guess this means we should probably have a EVP_ASYM_CIPHER-RSA also
even if it just points to other docs?

Reply:
We probably need to do a completeness check for all this documentation. But not within the scope of this PR.